processes/cve_management_process.md

```diff
@@ -0,0 +1,124 @@
+   been added to a closed mailing list that is used for announcements,
+   sharing documents, or discussion relevant to the CNA community.
+   The list rarely has more than ten messages a week.
+   **cna-discussion-list@iojs.org** 
+
+# CNA management processes
+
+## CVE Block management
+
+The CNA program allows the Node.js team to request a block of CVE's in
+advance. These CVE's are managed in an issue within the private Node.js
+security repo (https://github.com/nodejs/security).  Each year there
+will be an issue in that repo titled:
+
+```
+CVE Block for XXXX
+```
+
+were XXXX is the year (for example `CVE Block for 2017`).
+
+This issue will have the following sections:
+
+* Available
+* Pending
+* Announced 
+
+
+When a new block of CVEs  is received from Mitre they will be listed under
+the `Available` secion. 
+
+These CVE's will be moved from the Available to Pending and Announced
+as outlined in the section titled `CVE Management process`. 
+
+In addition, when moving a CVE from Available such that there are less
+than 2 remaning CVE's a new block must be requested as follows:
+
+* Use the Mitre request form https://cveform.mitre.org/ with the 
+  option `Request a Block of CVEs' to request a new block`.
+* The new block will be sent to the requestor through email.
+* Once the new block has been received, the requestor will add them
+  to the Available list.
+
+**Note**:  When making any changes in the issue, first add a comment
+indicating the change being made.
+
+
+## External CVE request process
+
+When a request for a CVE is received via the cve-request@iojs.org
+email alias the following process will be followed (likely updated
+after we get hacker one up and running).
+
+* Respond to the requestor indicating that we have the request
+  and will review soon.
+* Open an issue in the security repo for the request
+* Review the request.
+  * If a CVE is appropriate then assign the
+    CVE as outline in the section titled
+    `CVE Management processes for Node.js vulnerabilities` and
+    return the CVE number to the requestor (along with the request
+    to keep it confidential until the vulnerability is announced)
+  * If a CVE is not appropriate then respond to the requestor
+    with the details as to why.
+
+## Quarterly reporting
+
+* There is a requirement for quarterly reports to Mitre on CVE
+  activity.  Not sure of the specific requirements yet.  Will
+  add details on process once we've done the first one.
+
+
+# CVE Management process for Node.js vulnerabilities
+
+When the Node.js team is going announce a new vulnerability the
+following steps are used to to assign, announce and report a CVE.
+
+* Select the next CVE in the block available from the CNA process as
+  outlined in the section above. 
+* Move the CVE from the unassigned block, to the Pending section along
+  with a link to the issue in the security repo that is being used
+  to discuss the vulnerability.
+* As part of the security announcement process 
```

Comment:
Maybe just link this and drop the explicit URL.

Reply:
Done